Source/Core/DolphinQt/TAS/StickWidget.cpp

```diff
@@ -62,11 +62,13 @@ void StickWidget::paintEvent(QPaintEvent* event)
 void StickWidget::mousePressEvent(QMouseEvent* event)
 {
   handleMouseEvent(event);
+  m_ignore_movement = event->button() == Qt::RightButton;
 }
 
 void StickWidget::mouseMoveEvent(QMouseEvent* event)
 {
-  handleMouseEvent(event);
+  if (!m_ignore_movement)
```

Comment:
Should you reset this on mouse up? Otherwise it would gobble up plain move events (without press) as soon as it was used once with RMB.

Reply:
Plain move events are never used within the interface.